pkg/models/model.go

```diff
@@ -13,3 +25,57 @@ type ManagedModel struct {
 type NamespacedModel struct {
 	Namespace string `json:"-" yaml:"-"`
 }
+
+type Address struct {
+	Country     *string `json:"country"`
```

Comment:
I would add a country code too. I will refactor those to be enims based on gobl later.